src/pages/accountAndSettings/changeEmail.tsx

```diff
@@ -0,0 +1,73 @@
+import { LargeButton } from "@/components/LargeButton";
+import Head from "next/head";
+import { FiChevronLeft, FiUser } from "react-icons/fi";
+import { Transition } from "@headlessui/react";
+import { FormEvent, useState } from "react";
+import { signIn } from "next-auth/react";
+import Link from "next/link";
+
+type buttonProps = {
+	variant: "primary";
+	label: string;
+	disable: boolean;
+};
+
+export default function Settings(props: buttonProps) {
+	const [email, setEmail] = useState("");
+
+	const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+		e.preventDefault();
+		signIn("email", { email: email, callbackURL: "http://localhost:3000/" });
+	};
+	const emailCheck = new RegExp(`[A-z]+@[a-z]+.[a-zA-Z]{2,3}$`);
+
+	return (
+		<>
+			<Head>
+				<title>Create Next App</title>
+				<meta name="description" content="Generated by create next app" />
+				<meta name="viewport" content="width=device-width, initial-scale=1" />
+				<link rel="icon" href="/favicon.ico" />
+			</Head>
+			<div className="px-8 flex flex-col">
+				<header className="w-full flex items-center h-10 pt-8">
+					<Link href="/accountAndSettings">
+						<FiChevronLeft size={28} />
+					</Link>
+				</header>
+				<div className="w-full h-full flex-grow flex flex-col justify-around">
+					<div className="flex h-full w-full flex-col items-center gap-5">
```

Comment:
flexboxes already expand to their container, `h-full w-full` isn't necessary on html elements already with `flex`applied

Reply:
hehe that is totally true